src/ASM/LR/ASMu3Dmx.C

```diff
@@ -402,14 +404,14 @@ bool ASMu3Dmx::integrate (Integrand& integrand,
       if (vol < 0.0)
       {
         ok = false; // topology error (probably logic error)
-        break;
+        continue;
```

Comment:
In the equivalent tensor-spline method `ASMs3Dmx::integrate` we have `break` here instead of `continue`. Which one is correct? Same for the next two changes below (L412 and L481).

Reply:
we cannot break out of an openmp enabled loop. The loop nests are different for LR vs structured (one less loop as we have many colors that each can run in parallel instead of 2 colors with several strips/subgroups).